src/EditorFeatures/Core/Implementation/TodoComment/TodoCommentIncrementalAnalyzer.cs

```diff
@@ -51,7 +52,8 @@ public async Task AnalyzeSyntaxAsync(Document document, InvocationReasons reason
             // but, can be called concurrently for different documents in future if we choose to.
             Contract.ThrowIfFalse(document.IsFromPrimaryBranch());
 
-            if (!document.Project.Solution.Options.GetOption(InternalFeatureOnOffOptions.TodoComments))
+            if (!document.Project.Solution.Options.GetOption(InternalFeatureOnOffOptions.TodoComments) ||
+                ServiceFeatureOnOffOptions.IsPowerSaveModeEnabled(document.Project))
```

Comment:
hrmm... so what's the experience like with this sort of thing.  Consider if the user has existing todo comments up.  Then they enter powersaving mode. T hen they remove one.  The todo comment just stays up right?  This will appear as if things are broken.

Can we instead return an empty list and yellow-bar the todo window (or place a dummy item into it) saying that todo items are not being reported because of power-saving?